### ISSUE TITLE
Systems Manager Parameter Store - Incorrect ParameterArn

### ISSUE THREAD

**Issue by mshober** [aws/aws-cdk#4542]
The `parameterArn` property of both `StringParameter` & `StringListParameter` are incorrect

### Reproduction Steps
Given the following resources
```typescript
const stringParameter = new ssm.StringParameter(this, 'StringParameter', {
    parameterName: 'stringParameter',
    stringValue: 'value'
})

const stringListParameter = new ssm.StringListParameter(this, 'StringListParameter', {
    parameterName: 'StringListParameter',
    stringListValue: ['value']
})
```
Here are the incorrect arn values produced by `parameterArn`
```
arn:aws:ssm:region:account:parameterstringParameter
arn:aws:ssm:region:account:parameterStringListParameter
```
Expected values:
```
arn:aws:ssm:region:account:parameter/stringParameter
arn:aws:ssm:region:account:parameter/StringListParameter
```

### Fix
In `@aws-cdk/aws-ssm/lib/parameter.js`

Change
```typescript
function arnForParameterName(scope, parameterName) {
    return core_1.Stack.of(scope).formatArn({
        service: 'ssm',
        resource: 'parameter',
        sep: '',
        resourceName: parameterName,
    });
}
```
To
```typescript
function arnForParameterName(scope, parameterName) {
    return core_1.Stack.of(scope).formatArn({
        service: 'ssm',
        resource: 'parameter',
        sep: '/',
        resourceName: parameterName,
    });
}
```
### Environment

  - **CLI Version      : 1.13.1**
  - **Framework Version: 1.13.1**
  - **Language         : Typescript**

--- 

This is :bug: Bug Report

**Comment by nmussy** [comment]
@eladb Did you fix this with #4685?

**Comment by eladb** [comment]
Yes, this is fixed now.
